apiary.apib

```diff
@@ -766,9 +766,86 @@ This operations is available only for organization administrators.
     + Body
 
             {
+                "uri": "https://connection.keboola.com/v2/storage/buckets/in.c-my-bucket",
+                "id": "in.c-my-bucket",
+                "name": "c-my-bucket",
+                "stage": "in",
+                "description": "Some Description",
+                "tables": "https://connection.keboola.com/v2/storage/buckets/in.c-my-bucket/tables",
+                "created": "2017-02-13T12:01:05+0100",
+                "lastChangeDate": null,
+                "isReadOnly": false,
+                "dataSizeBytes": 0,
+                "rowsCount": 0,
+                "isMaintenance": false,
+                "backend": "snowflake",
+                "sharing": "organization"
+            }
+
+### Share Bucket To Organization [POST /v2/storage/buckets/{bucket_id}/share-organization]
+[Enables sharing](https://help.keboola.com/storage/buckets/sharing/) or **update sharing** of a bucket.
+Bucket is shared to organization. Any organization member can link bucket to project in organization.
+This operations is available only for organization administrators.
```

Comment:
Ok, I think this is better again, sorry:
`Enable sharing for this bucket within the organization.  Any organization member will be able to link to this bucket from any project within the organization.
This operations is available only for organization administrators.`